packages/vscode-extension/src/controls/webviewPanel.ts

```diff
@@ -239,9 +241,11 @@ export class WebviewPanel {
       return Promise.resolve();
     };
 
-    AppStudioTokenInstance.setStatusChangeMap("quick-start-webview", m365WebviewCallback);
-    SharepointTokenInstance.setStatusChangeMap("quick-start-webview", m365WebviewCallback);
-    GraphTokenInstance.setStatusChangeMap("quick-start-webview", m365WebviewCallback);
+    M365TokenInstance.setStatusChangeMap(
```

Comment:
One question may not related to this PR but related to login - is it possible that m365 login is triggered by graph/sharepoint scopes? Will that refresh treeview/webview as well?

Reply:
Yes, they all will trigger `notifyStatus` event.